src/languages/kotlin.js

```diff
@@ -9,8 +9,11 @@ function(hljs) {
     keyword:
       'abstract as val var vararg get set class object open private protected public noinline ' +
       'crossinline dynamic final enum if else do while for when throw try catch finally ' +
-      'import package is in fun override companion reified inline lateinit init' +
+      'import package is in fun override companion reified inline lateinit init ' +
       'interface annotation data sealed internal infix operator out by constructor super ' +
+      'tailrec where const inner ' +
+      // new from kotlin 1.1 & 1.2
```

Comment:
I think this comment is unnecessary. We do not classify keywords by which language version introduced them.

Reply:
I marked this just for the contributor later to tell these keywords apart more easily. I would move it if necessary.